engine/src/AnimatedSpritesUpdater.cpp

```diff
@@ -0,0 +1,25 @@
+//
+// Created by tdancois on 8/4/16.
+//
+
+#include "AnimatedSpritesUpdater.h"
+
+AnimatedSpritesUpdater *AnimatedSpritesUpdater::instance;
+
+AnimatedSpritesUpdater::AnimatedSpritesUpdater() {}
+
+AnimatedSpritesUpdater *AnimatedSpritesUpdater::getInstance() {
```

Comment:
Use something like
```c++
AnimatedSpritesUpdater& AnimatedSpritesUpdater::getInstance() {
    static AnimatedSpritesUpdater instance;
    return instance;
}
```

Because you don't have to care about memory leaks due to a forgotten `deleteInstance()`